tests/Doctrine/Tests/Models/Rating/Reviews.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\Rating;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+class Reviews extends ArrayCollection
```

Comment:
Given that the ORM does not support using custom collection types when loading the entity, I suggest avoiding to create this class as it would show a bad practice.